### PR TITLE
ci: make optimzie health status workflow work in monorepo

### PR DESCRIPTION
**Pull request by misiekhardcore**
## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

relates to https://github.com/camunda/camunda-optimize/issues/13914

### CODE REVIEW

File: .github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts [hunk 1]
```diff
@@ -40,68 +35,23 @@ async function createHealthStatusConfig() {
...
-        'optimize-import-mediator-permutation',
-        'optimize-history-cleanup-performance',
-        'optimize-connect-to-secured-es',
-        'optimize-cluster-test',
-        'optimize-e2e-tests',
-        'optimize-release-optimize',
-        'optimize-trivy-check',
-      ],
-    },
-    githubPrs: {
```
Comment (misiekhardcore) [inline .github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts:70]: I don't think there is a list of deps PRs only related to Optimize as we are providing those updates for the whole monorepo, so I delete this whole section

File: .github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts [hunk 2]
```diff
@@ -5,32 +5,27 @@
...
+  const defaultBranch = 'main';
+  const releaseBranches = await githubService.getBranchesWithPrefix('release/optimize-');
+  const stableBranches = await githubService.getBranchesWithPrefix('stable');
+  const ciBranches = [defaultBranch, ...releaseBranches, ...stableBranches].sort(
     githubService.sortBranches,
   );
-  const maintenanceBranches = ciBranches.filter((branch) => branch.includes('maintenance'));
 
   const config: Partial<Config> = {
     argoCd: {
```
Comment (misiekhardcore) [inline .github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts:31]: Should we also remove ArgoCD from here?

File: .github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts [hunk 3]
```diff
@@ -5,103 +5,46 @@
...
 
 const CONFIG_FILE_NAME = 'config.json';
 
 async function createHealthStatusConfig() {
   const githubService = new GitHubService(GITHUB_TOKEN, GITHUB_ORG, GITHUB_REPO);
-
-  const renovateStringBranches = getRenovateStringBranches();
-  const releaseBranches = await githubService.getBranchesWithPrefix('release');
-  const ciBranches = [...releaseBranches, ...renovateStringBranches].sort(
+  const defaultBranch = 'main';
```
Comment (OmranAbazid) [inline .github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts:20]: I sugges to move this up and make it global

**Review by OmranAbazid — APPROVED** [review]

File: .github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts (hunk 2 above)
Comment (misiekhardcore) [reply .github/optimize/scripts/healthStatusReport/createHealthStatusConfig.ts]: as per this [convo](https://camunda.slack.com/archives/C04FM0DRJT0/p1723544853899339?thread_ts=1723544541.548459&cid=C04FM0DRJT0), we remove argo

**Pull request merged**